client/src/components/item/ItemEdit.tsx

```diff
@@ -1,47 +1,47 @@
-import React, {ChangeEvent, Component, FormEvent} from 'react';
+import React, {ChangeEvent, Component, FormEvent} from "react";
 import {match} from "react-router";
 import {Item} from "../inventory/HardwareItem";
 import {Button, CheckboxProps, DropdownProps, Form, Label, Popup} from "semantic-ui-react";
 import AddOptionDropdown from "./AddOptionDropdown";
 
 interface ItemDetails {
-    requireApproval: boolean,
-    returnRequired: boolean,
-    price: number,
-    hidden: boolean,
-    owner: string,
+    requireApproval: boolean;
+    returnRequired: boolean;
+    price: number;
+    hidden: boolean;
+    owner: string;
 }
 
 interface ItemEditProps {
-    match: match & ItemEditParams
+    match: match & ItemEditParams;
 }
 
 interface ItemEditParams {
-    params: { itemId: string }
+    params: { itemId: string };
 }
 
-export type ItemComplete = Item & ItemDetails
+export type ItemComplete = Item & ItemDetails;
 
 interface ItemEditState {
-    loading: boolean,
-    item: ItemComplete | null,
+    loading: boolean;
+    item: ItemComplete | null;
 
-    [name: string]: any | any[],
+    [name: string]: any | any[];
 }
 
 class ItemEdit extends Component<ItemEditProps, ItemEditState> {
     constructor(props: ItemEditProps) {
         super(props);
         this.state = {
-            loading: true,
-            item: null,
+            "loading": true,
+            "item": null,
             "item-return-required": true,
             "item-approval-required": true,
             "item-hidden": false,
-            itemOwnerChoices: [
-                {key: 'hackgt', text: 'HackGT', value: 'HackGT'},
-                {key: 'hive', text: 'The Hive', value: 'The Hive'},
-                {key: 'is', text: 'Invention Studio', value: 'Invention Studio'}]
+            "itemOwnerChoices": [
```

Comment:
can you extract these to a config file and import it instead
you can do this in a separate PR if you're fed up with this one

Reply:
It might need to be structured like categories TBH with a separate DB table tbh

Reply:
Probably separate PR material.  This is strongly tied to a separate discussion of whether we need to handle items that live in different locations and having multiple hardware desks (e.g., ours + one in The Hive), and item owner is kinda a surrogate for a proper Location field.

Reply:
:ok: